content/en/docs/distributions/gke/deploy/overview.md

```diff
@@ -37,17 +36,17 @@ Once you finish deployment, you will be able to:
 1. get a [Cloud Endpoint](https://cloud.google.com/endpoints/docs) which is accessible via [IAP (Identity-aware Proxy)](https://cloud.google.com/iap).
 1. enable [Multi-user feature](/docs/components/multi-tenancy/) for resource and access isolation.
 1. take advantage of GKE's
-  [Cluster Autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler) 
-  to automatically resize the number of nodes in a node pool.
+   [Cluster Autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler)
+   to automatically resize the number of nodes in a node pool.
 1. choose GPUs and [Cloud TPU](https://cloud.google.com/tpu/) to accelerate your workload.
 1. use [Cloud Logging](https://cloud.google.com/logging/docs/) to help debugging and troubleshooting.
 1. access to many managed services offered by Google Cloud.
 
-  <img src="/docs/images/gke/full-kf-home.png" 
+<img src="/docs/images/gke/full-kf-home.png" 
     alt="Full Kubeflow Central Dashboard"
     class="mt-3 mb-3 border border-info rounded">
 
 ## Next steps
 
-* Repeat [Deploy Kubeflow Cluster](/docs/distributions/gke/deploy/deploy-cli/) if you want to deploy multiple clusters.
-* Run a full ML workflow on Kubeflow, using the [end-to-end MNIST tutorial](/docs/distributions/gke/gcp-e2e/).
+- Repeat [Deploy Kubeflow Cluster](/docs/distributions/gke/deploy/deploy-cli/) if you want to deploy multiple clusters.
+- Run a full ML workflow on Kubeflow, using the [end-to-end MNIST tutorial](/docs/distributions/ibm/iks-e2e/#run-the-mnist-tutorial-on-iks).
```

Comment:
Thank you for your change, let's use this link instead: https://github.com/kubeflow/pipelines/blob/e42d9d2609369b96973c821dca11fe5b2565e705/samples/contrib/kubeflow-e2e-mnist/kubeflow-e2e-mnist.ipynb

Reply:
![end-to-end MNIST tutorial](https://user-images.githubusercontent.com/76856697/177317687-57db4446-7646-4183-845d-3979215d0587.png)

thanks link updated